### DISCUSSION TITLE
🚀 GitHub Education has shipped Community Exchange on Global Campus.

### DISCUSSION THREAD

**Discussion by ettaboyle** [community/community#18194] [GitHub Education] [answered]
### TLDR
- 🚀 GitHub Education has shipped [Community Exchange](https://github.blog/2022-06-08-start-your-open-source-journey-with-community-exchange/) on [Global Campus](https://github.blog/2021-09-01-introducing-github-global-campus/).
- 🧑‍🎓 Community Exchange is a safe place for students to submit and discover student repositories, learn, and support each other.
- 🏫 Community Exchange promotes collaboration and learning amongst the 2 million students within Global Campus.
- 📂 We believe Community Exchange will make contributing to an open source repository less stressful for students

![172778114-de660c98-001f-43d3-9c81-8f58522bcdd3](https://user-images.githubusercontent.com/75342807/172959162-265996f6-75a9-43e5-a1c1-9afaf48defe5.png)

**Comment by ettaboyle** [comment, marked as answer]
🔗 [Read more details on the GitHub Blog](https://thehub.github.com/news/2022-06-09-community-exchange/)
